keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -478,6 +482,47 @@ func retryDownload(filename, checksumFilename string, version semver.Version, ta
 	return fmt.Errorf("failed to download %s", filename)
 }
 
+func download(downloadURL, targetPath string) error {
+	name := filepath.Base(downloadURL)
+	resp, err := http.Get(downloadURL)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	if targetPath != "" {
+		_, err = os.Stat(targetPath)
+		if err != nil {
+			mkErr := os.MkdirAll(targetPath, os.ModePerm)
+			if mkErr != nil {
+				return mkErr
+			}
+		}
+	}
+	fullPath := filepath.Join(targetPath, name)
```

Comment:
if targetPath = "", targetPath  will be default value?

Reply:
if targetPath = "", the path will be the current directory